infra/modules/kubernetes/README.md

```diff
@@ -11,10 +11,9 @@
 | k8s\_namespace\_name | kubernetes namespace name | `string` | n/a | yes |
 | k8s\_service\_account\_name | kubernetes service account for pods | `string` | n/a | yes |
 | labels | A map of key/value label pairs to assign to the resources. | `map(string)` | `{}` | no |
-| network\_self\_link | The VPC network self\_link to host the k8s cluster | `string` | n/a | yes |
 | project\_id | GCP project ID. | `string` | n/a | yes |
 | region | The region to host the k8s cluster | `string` | n/a | yes |
-| zones | The zones to host the k8s cluster | `list(string)` | n/a | yes |
+| zones | The zone to host the k8s cluster | `list(string)` | n/a | yes |
```

Comment:
**Just a heads-up:** No action needed. 
Not sure if you also wanted to rename this to `zone` (from `zones`) might.

Reply:
Done